src/ext/NetFxExtension/wixlib/NetFx461.wxs

```diff
@@ -0,0 +1,106 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+  <copyright file="NetFx461.wxs" company="Outercurve Foundation">
+    Copyright (c) 2004, Outercurve Foundation.
+    This software is released under Microsoft Reciprocal License (MS-RL).
+    The license and further copyright text can be found in the file
+    LICENSE.TXT at the root directory of the distribution.
+  </copyright>
+-->
+<Wix xmlns="http://schemas.microsoft.com/wix/2006/wi" xmlns:util="http://schemas.microsoft.com/wix/UtilExtension">
+
+  <!--
+        .NET Framework installation state properties
+
+        Official documentation can be found at the following location:
+
+           .NET Framework 4.5/4.5.1/4.5.2/4.6/4.6.1 - http://msdn.microsoft.com/en-us/library/w0x726c2(v=vs.110).aspx
+    -->
+
+  <?define NetFx461MinRelease = 394254 ?>
+  <?define NetFx461WebLink = http://go.microsoft.com/fwlink/?LinkId=671728 ?>
+  <?define NetFx461RedistLink = http://go.microsoft.com/fwlink/?LinkId=671743 ?>
+  <?define NetFx461EulaLink = https://msdn.microsoft.com/en-us/library/ee942965.aspx ?>
```

Comment:
The EULA link needs to be a direct link to the EULA, not a .NET information page.